mabl snippets/native mobile testing/configurableDurationLongPressOnElement.js

```diff
@@ -0,0 +1,47 @@
+function mablJavaScriptStep(
+  mablInputs,
+  callback,
+  duration = "1000",
+  xpath = undefined
+) {
+  // Call the Appium function
+  callAppium(xpath, duration)
```

Comment:
Should the `callAppium` be more specific to each snippet or did you leave it like this for the mobile snippets on purpose?

Reply:
@simonychoy I was thinking about making it more specific, but I kinda liked showing that can be the common structure. I had it named uniquely each time and switched to making it the same. I am very open to your thoughts though!